dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

```diff
@@ -1170,6 +1170,8 @@ dot.common.message.unlocking=Unlocking...
 dot.common.select.workflow=Select Workflow
 dot.common.select.workflows=Select Workflows
 dot.common.whats.changed=What's Changed?
+dot.template.builder.add.row=Add Row
```

Comment:
Can you add "Add Container" and all the other text that needs to be translated in the template builder lib.

Reply:
Done!